llvm/include/llvm/Analysis/StructuralHash.h

```diff
@@ -13,15 +13,18 @@
 
 namespace llvm {
 
+enum class StructuralHashOptions { None, Detailed, CallTargetIgnored };
```

Comment:
Can you document these options and include their string values, e.g., `call-target-ignored`?